Jenkinsfile

```diff
@@ -47,22 +40,22 @@ node {
   }
 }
 
-def godep(repo) {
+def gofmt() {
   docker.withRegistry('https://registry.internal.exoscale.ch') {
-    def image = docker.image('registry.internal.exoscale.ch/exoscale/golang:1.10')
+    def image = docker.image('registry.internal.exoscale.ch/exoscale/golang:1.11')
     image.pull()
-    image.inside("-u root --net=host -v ${env.WORKSPACE}/src:/go/src/github.com/${repo}") {
-      sh 'test `gofmt -s -d -e . | tee -a /dev/fd/2 | wc -l` -eq 0'
-      sh "cd /go/src/github.com/${repo} && dep ensure -v -vendor-only"
+    image.inside("-u root --net=host") {
+      sh 'test $(gofmt -s -d -e $(find -iname "*.go" | grep -v "/vendor/") | tee -a /dev/fd/2 | wc -l) -eq 0'
     }
   }
 }
 
 def golint(repo, ...extras) {
   docker.withRegistry('https://registry.internal.exoscale.ch') {
     def image = docker.image('registry.internal.exoscale.ch/exoscale/golang:1.10')
+    image.pull()
     image.inside("-u root --net=host -v ${env.WORKSPACE}/src:/go/src/github.com/${repo}") {
-      sh "golint -set_exit_status -min_confidence 0.6 `go list github.com/${repo}/... | grep -v /vendor/`"
+      sh "golint -set_exit_status -min_confidence 0.3  `go list github.com/${repo}/... | grep -v /vendor/`"
```

Comment:
Could be `golint -set_exit_status -min_confidence 0.3 ./...` (I think, but nonetheless, it has been several Go versions that `./...` doesn't match vendor anymore).